command/acl_bootstrap_test.go

```diff
@@ -76,3 +78,67 @@ func TestACLBootstrapCommand_NonACLServer(t *testing.T) {
 	out := ui.OutputWriter.String()
 	assert.NotContains(out, "Secret ID")
 }
+
+// Attempting to bootstrap the server with an operator provided token should
+// return the same token in the result.
+func TestACLBootstrapCommand_WithOperatorBootstrapToken(t *testing.T) {
+	ci.Parallel(t)
+	assert := assert.New(t)
```

Comment:
We're trying to remove this aliasing from the code base (same for the tests below). I thought it was picked up by a lint though, so that may need to be tweaked.

```suggestion
```